exercises/practice/bob/.approaches/introduction.md

```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+There are various idiomatic approaches to solve Bob.
+A basic approach can use a series of `if` statements to test the conditions.
+Or a [switch][switch] on a [tuple][tuple] of the conditions can be used.
+An array can contain answers from which the right response is selected by an index calculated from scores given to the conditions.
+
+# General guidance
+
+Regardless of the approach used, some things to look for include
```

Comment:
@kytrinyx @iHiD I was wondering if you have thoughts on how we should be addressing people in this document. Should we make this impersonal and be about "solutions should ensure that:", or something more personal like what we have here that seems to be addressing a student more directly.

The approaches I've written we're all very matter-of-factly, less personal.

Reply:
I like the personal approach. I'd maybe go even further
> Regardless of the approach used, some things you could look out for include